docs/versioning/how-to-version-in-data-store.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+nav_order: 4
+title: How to create versions (Data Store) 
+has_children: false
+parent: Versioning
+---
+# How to create versions in the Data Store
+
+## Step 1. Navigate to the dataset record in the Data Store
+
+- Ensure you're logged in.
+- Navigate to the dataset record
+- Click on the "VERSIONING" tab as shown below
+
+|                         Versioning Tab                                                                 |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-tab.png" alt="drawing" width="600"/> |
+
+
+
+## Step 2. Create a new version
```

Comment:
Need to add a catch here for if the user is not on the latest version OR they don't have registry write permissions OR they don't have resource level ADMIN access - all of these will make the button not appear/be greyed out(?).

Reply:
perhaps we need to add a "pre-requisites" before this